test/01LessonOneTest.js

```diff
@@ -5,36 +5,39 @@ var assert = require('assert');
 
 function returnTrue() {
     // should return true
-
+    return true;
 }
 
 function returnFalse() {
     // should return false
-
+    return false;
 }
 
 function greaterThan(big, small) {
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
-
+    return 4 > 3;
+    console.log(4 < 3);
```

Comment:
@wlittler `big` and `small` are the values that are passed in to the function. You should be comparing `big` and `small`.

Like this:
```
 function greaterThan(big, small) {
      // should return true if the first argument is greater than the second argument
      // and return false if the first argument is less than the second 
      return big > small;
}
```